tests/simulation_sanity_test/simulation_test.py

```diff
@@ -0,0 +1,27 @@
+import json
+import warnings
+
+from consts import Consts
+from manager import SimulationManager
+
+f = open('simulation_test_ranges.json', 'r')
+range_per_day_dict = json.load(f)
+
+
+def test_no_policy_simulation():
+    """
+    run the simulation with no policy and fixed R0
+    """
+    warnings.warn(UserWarning("Ranges for each state was generate from real sim runs, might be flaky."))
+    consts = Consts(active_isolation=False, population_size=1_000, r0=2.4)
```

Comment:
There are many default consts that might change and mess with the results. 
Maybe you should export the consts to an external file and load it in the test

Reply:
done